djangomodelimport/fields.py

```diff
@@ -77,7 +78,7 @@ def to_python(self, value):
         value = (value or '').strip()
         if value:
             try:
-                return from_current_timezone(parser.parse(value))
+                return from_current_timezone(parser.parse(value, dayfirst=bool(match(r'^\d{2}.\d{2}.\d{4}$',value))))
```

Comment:
Maybe `\d\d?` is better than `\d{2}`. `\d{4}` is req'd for obvious reasons so keep that one.